pom.xml

```diff
@@ -2232,6 +2236,19 @@
         <version>${version.org.w3c.css.sac}</version>
       </dependency>
 
+      <!-- libraries for incremental compilation-->
+      <dependency>
+        <groupId>commons-cli</groupId>
+        <artifactId>commons-cli</artifactId>
+        <version>${version.commons.cli}</version>
+      </dependency>
```

Comment:
Sorry for nit-picking, but this is technically not needed. Overriding the version property is enough (as the kie-parent directly inherits from ip-bom).